.travis/install_saw.sh

```diff
@@ -32,7 +32,9 @@ mkdir -p "$DOWNLOAD_DIR"
 cd "$DOWNLOAD_DIR"
 
 #download saw binaries
-curl --retry 3 https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2017-07-27-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
+##TODO: Post the SAW binary here once we've tested this
+#curl --retry 3 https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2017-07-27-Ubuntu14.04-64.tar.gz --output saw.tar.gz;
+curl https://saw.galois.com/builds/nightly/saw-0.2-2018-01-21-Ubuntu14.04-64.tar.gz > saw.tar.gz;
```

Comment:
Uploaded it to our S3 bucket. Can you update the link to `https://s3-us-west-2.amazonaws.com/s2n-public-test-dependencies/saw-0.2-2018-01-21-Ubuntu14.04-64.tar.gz` ?

Reply:
done